### DISCUSSION TITLE
[Octernships] Which branch should the changes be done?

### DISCUSSION THREAD

**Discussion by naveen521kk** [community/community#49620] [GitHub Education] [answered]
Hey, I've recently applied for an Octernship and I'm confused about which branch I should be working on. 
> Students are expected to use the [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) when working on their project.
>  1. Making changes on the auto generated feedback branch to complete the task.
>  2. Using the auto generated Feedback Pull Request for review and submission
>  3. Using GitHub Discussions to ask any relevant questions regarding the project

These instructions (from the repo) in my understanding tell that I should be working on the `feedback` branch. The confusion I got when looking at the auto-generated "Feedback Pull Request" is that the bot created it to merge into `feedback` the changes from `main`, `main -> feedback`, this is as if I should be working on the `main` branch and it'll be merged into the `feedback` branch on review. This is the other way around, actually.

![firefox_KaqDnPZuIE](https://user-images.githubusercontent.com/49693820/224028504-4c7fb3c9-77a4-4770-90e8-c338db69a507.png)
I'm confused about this and would like some assistance. Thanks!

**Comment by samus-aran** [comment, marked as answer]
Hi @naveen521kk 

Looks like you got [your answer one of the partners' repo](https://github.com/HousewareHQ/frontend-engineering-octernship/issues/6#issuecomment-1467401306). 👍
